UIKitTemplate/UIKitTemplate/ChooseSizeViewController.swift

```diff
@@ -0,0 +1,112 @@
+// ChooseSizeViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+protocol SizeDelegate: AnyObject {
+    /// передача размера обуви в корзину
+    func sendSize(size: Int)
```

Comment:
должны быть задокументирвоано, что за параметры передаются
```suggestion
    /// что делает
    /// - Parameters:
    ///    - size: описание
    func sendSize(size: Int)
```